src/service/resource.js

```diff
@@ -441,12 +402,7 @@ export class Resource {
    */
   completeCollapse() {
     toggle(this.element, false);
-    this.layoutBox_ = layoutRectLtwh(
-        this.layoutBox_.left,
-        this.layoutBox_.top,
-        0, 0);
-    this.isFixed_ = false;
-    this.element.updateLayoutBox(this.layoutBox_);
+    this.requestMeasure();
```

Comment:
I don't see an issue. A pass will always be scheduled, it's just how long it has to wait before it happens.